calico_versioned_docs/version-3.30/variables.js

```diff
@@ -37,9 +37,18 @@ const variables = {
     flannelMigration: 'calico/flannel-migration-controller',
     'calico/dikastes': 'calico/dikastes',
     'pilot-webhook': 'calico/pilot-webhook',
-    flexvol: 'calico/pod2daemon-flexvol',
+    flexvol: 'flexvol',
```

Comment:
Still not sure why we need both flexvol and `pod2daemon-flexvol` here - any idea?

Reply:
I don't know either. I'm just repeating what is see in `calico_versioned_docs/version-3.30/releases.json`. But that may be a manual error. 

@radTuti Any chance you know about this?

Reply:
current changes sound good